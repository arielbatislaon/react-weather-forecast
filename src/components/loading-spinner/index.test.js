import React from 'react';
import renderer from 'react-test-renderer';

import LoadingSpinner from './index';

it('renders LoadingSpinner correctly ', () => {
  const tree = renderer.create(<LoadingSpinner />).toJSON();
  expect(tree).toMatchSnapshot();
});