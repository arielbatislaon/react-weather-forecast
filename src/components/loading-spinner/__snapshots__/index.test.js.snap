// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders LoadingSpinner correctly  1`] = `
<div>
  <button
    className="btn btn-primary"
    disabled={true}
    type="button"
  >
    <span
      aria-hidden="true"
      className="spinner-grow spinner-grow-sm"
      role="status"
    />
    Loading...
  </button>
</div>
`;
